tech.json

```diff
@@ -898,6 +898,15 @@
                   ]
                 }
               ]
+            },
+            {
+              "name": "canDownwardGModeSetup",
+              "requires": [ "canEnterRMode" ],
+              "note": [
+                "Ability to setup an R-mode or G-mode through a downward door.",
+                "Samus needs to be in a standing or walking position with no vertical speed on the first frame in the next room, while taking damage through the transition.",
+                "This is typically done using quicksand, clipping into the floor, or using a thawing frozen wall crawler."
```

Comment:
Pretty sure any type of frozen enemy could work.  Covern, Atomic, a Gamet would be worth testing.

Reply:
True. I would think any enemy would work, probably anything besides a crawler will end up notable tho. I am particularly curious about the covern/atomic though